Dockerfile

```diff
@@ -21,9 +21,9 @@ EXPOSE 8080 8081
 CMD exec java $JAVA_OPTS \
      -Ddw.server.applicationConnectors[0].port=8080 \
      -Ddw.server.adminConnectors[0].port=8081 \
-     -Ddw.documentStoreApi.endpointConfiguration.primaryNodes=$VULCAN_HOST \
```

Comment:
I'm assuming you will also update the service file as part of https://github.com/Financial-Times/up-service-files/pull/1372

Reply:
Done. https://github.com/Financial-Times/up-service-files/pull/1372/commits/4af68e993d8652404b4f90d03bf165fdfb5d5624